### ISSUE TITLE
Can't upload video to avatar

### ISSUE THREAD

**Issue by Neurotoxin001** [telegramdesktop/tdesktop#8304]
I just can't chose video for avatar:
![image](https://user-images.githubusercontent.com/39812401/88476497-8a663780-cf41-11ea-9159-edaf1f56199d.png)
When I chose "all files" and select mp4:
![image](https://user-images.githubusercontent.com/39812401/88476500-8c2ffb00-cf41-11ea-95ba-a6aff1870e03.png)

### Configuration
**Operating system:** Win 10 LTSC 1809

**Version of Telegram Desktop:** 2.2

**Used theme**: Dark default

**Comment by john-preston** [comment]
@Neurotoxin001 Setting video as avatar is not supported in tdesktop :(

**Comment by amavlyanov** [comment]
> @Neurotoxin001 Setting video as avatar is not supported in tdesktop :(

that's exactly what is this issue about!

**Comment by Neurotoxin001** [comment]
And downloading video avatars as video is not supported too :/

**Comment by Neurotoxin001** [comment]
Still needed

**Comment by Neurotoxin001** [comment]
Wtf with this bot? I got same notification for this issue 5 times

![image](https://user-images.githubusercontent.com/39812401/96998288-de31ee00-153b-11eb-8292-d5130a3455f7.png)

**Comment by AhyaH-YHWH** [comment]
> @Neurotoxin001 Setting video as avatar is not supported in tdesktop :( @john-preston 

Can't desktop just set square video/Gif as profile video
Checking the video is square, under 10s, smaller than 2MB and have smaller size of 800*800 pixel
If possible mute the video that has sound
(since all these details are included in mp4 files properties)
if the video doesn't have requirement, send an error this video is not square or larger than 800*800 and...

![image](https://user-images.githubusercontent.com/73444508/107738601-3cb5ee80-6d1c-11eb-9801-835d7be8230f.png)

**Comment by Spidey611** [comment]
@AhyaH-YHWH its still not working

**Comment by Neurotoxin001** [comment]
> And downloading video avatars as video is not supported too :/

Downloading video avatars is now supported. But not uploading.

![image](https://user-images.githubusercontent.com/39812401/174185681-9cc0529d-8e2b-4a12-a767-345664313e07.png)

**Comment by goosegit11** [comment]
still needed.

**Comment by Neurotoxin001** [comment]
Does this mean that we finally can upload video avatars?
https://github.com/telegramdesktop/tdesktop/commit/9513aaa768c250b4275865a97e2e15943676bdcc

**Comment by Neurotoxin001** [comment]
It's still not supported in 4.4.2 beta…
![image](https://user-images.githubusercontent.com/39812401/209829100-bf4b5daf-c5b6-42b3-8886-3ce7edbb3e22.png)

And I can't suggest video avatar too. Only these formats is supported:

![image](https://user-images.githubusercontent.com/39812401/209829240-c4fae6ae-5711-4c31-9bb4-daa6b87f70d4.png)

**Comment by john-preston** [comment]
Yes, sorry. This commit was about accepting video avatars suggested from mobile apps, if they allow it.

**Comment by TadavomnisT** [comment]
@john-preston 

It's quit some time since Telegram added this feature (Video as avatar), 2 years I think!
What's stopping tdesktop from supporting that feature?

Is there any complexity in processing video and prepare it for limitations?
If so, we desktop folks really can appreciate a simple uploader... no need to process the video for now! Just upload it and if it fails a pop-up can say:

________________________________________
**Can't use the video, The profile video limitations are:**
* The file format must be a `.mp4` video (not `.gif` file).
* The video dimensions must be 800x800 pixels.
* The maximum video size must be less than 2MB.
* The maximum video duration must be less than 10 seconds.
________________________________________


_This information was originally [posted in Telegram Beta Chat](https://t.me/tgbetachat/325956), later [mentioned in Telethon Chat](https://t.me/TelethonChat/235560)._

I think this is what most tdlib wrappers do!
We really can use a hand in here mate... Thanks!

**Comment by TodoshiTakato** [comment]
still needed

**Comment by VinceHov** [comment]
still needed

**Comment by torturado** [comment]
guys use mobile app, it works there

**Comment by Neurotoxin001** [comment]
> guys use mobile app, it works there

We know, but we need that in a desktop app too. It's been five years, after all.

**Comment by perdakovich** [comment]
sneeded

**Comment by werod45** [comment]
still need that

**Comment by mammuthus** [comment]
Olympiad kids can’t fix bugs that have existed for years, let alone ensure consistency of functionality between client versions…
